Source/diablo.cpp

```diff
@@ -1,6 +1,62 @@
-#include "diablo.h"
+#include "../types.h"
 #include "../3rdParty/Storm/Source/storm.h"
 #include "../DiabloUI/diabloui.h"
+#include "diablo.h"

```

Comment:
Wow, some of these files have a truly massive amount of imports.

Reply:
Indeed ^^